css-pseudo-classes/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link href="styles.css" type="text/css" rel="stylesheet" />
+  <title> CSS PSEUDO CLASSES </title>
+</head>
+
+<body>
+  <h2> Hover Button </h2>
+  <form>
+    <label>
+      <button class="click" type="button"> I'm a button! </button>
+    </label>
+  </form>
```

Comment:
No need for a `form`, because we aren't collecting / submitting any data. There are no `input` elements. This should just be a `button` element, with no `label` attached.